types/pouchdb-upsert/index.d.ts

```diff
@@ -1,6 +1,9 @@
-// Type definitions for pouchdb-upsert 2.2
+// Type definitions for pouchdb-upsert 2.3
```

Comment:
Can you roll back the version? We try to correspond to MAJOR.MINOR on definitelytyped